include/sot/core/device.hh

```diff
@@ -40,15 +40,15 @@ namespace dynamicgraph {
     /// Define the type of input expected by the robot
     enum ControlInput
     {
-      CONTROL_INPUT_POSITION=0,
-      CONTROL_INPUT_VELOCITY=1,
-      CONTROL_INPUT_ACCELERATION=2,
+      CONTROL_INPUT_NO_INTEGRATION=0,
+      CONTROL_INPUT_ONE_INTEGRATION=1,
```

Comment:
> Theoretically a direct dynamic algorithm like CRBA would provide you position based on torque.
Do you mean ABA?
I don't think we need to implement the full integration scheme for torque control, unless we are implementing a simulation device in SoT (which I think Andrea did for device control). In the device class, the requirement would be just to populate the outgoing signals.